pytorch_blade/src/compiler/jit/torch/op_registry.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
```

Comment:
do we need any unit test?

Reply:
This pull request only adds source files and makes compilation successfully. Unit tests can be added later.

Reply:
better to have an issue to trace that.